src/g3log/logworker.hpp

```diff
@@ -109,6 +109,13 @@ namespace g3 {
          removeWrappedSink(sink_handle->sink());
       }
 
+      void flush(bool propogate_to_sinks = true) {
+         g3::spawn_task([propogate_to_sinks] {
+            if (propogate_to_sinks) {
+               // TODO: ...

```

Comment:
Is this a placeholder for whether or not the flush should be triggering on all sinks as well?